python/mujinplanningclient/realtimerobotplanningclient.py

```diff
@@ -523,7 +700,7 @@ def UpdateLocationContainerIdType(self, locationName, containerName, containerId
             containerName (str): Name of the container
             containerId (str): ID of the container
             containerType (str): Type of the container
-            trackingCycleIndex (optional): If specified, then the cycle with same cycleIndex will update location tracking in the same call.
+            trackingCycleIndex: If specified, then the cycle with same cycleIndex will update location tracking in the same call.
```

Comment:
This should still be optional

Reply:
So we either always document (optional) when type is not specified or not. The master clients are inconsistent here. There are much more places where isn't omitted. And I'm not sure its a good idea to add it. It looks confusing, as if optional - is actually some special type.

Reply:
I don't think the word "optional" will be mistaken for a type, if that's your concern?

Reply:
Well, this (maybe noone will get confused, but it just looks wrong to me) and the fact that adding those creates much more diffs than in removes. It will be removed in 4 places while if we add - dozens across different clients.

PS: as I mentioned earlier - Im convinced that after type annotations are added these docstring types should be removed at all anyway.